### DISCUSSION TITLE
How can I disable a shim so that the native command is used?

### DISCUSSION THREAD

**Discussion by eggbean** [ScoopInstaller/Scoop#5600] [Help and Questions] [answered]
The Linux tools cannot read UTF-16 encoded files, it seems. When I try printing a `desktop.ini` file using `cat`, for example, I get this error:

    cat: desktop.ini: Windows stdio in console mode does not support writing non-UTF-8 byte sequences

How can I  can't use what I have shimmed to `less` either, so I want to use the standard Windows `more` command, but I have the `uutils-coreutils` package installed and that has a `more` command.

How can I disable this so that the native `more` command is used?

**Comment by HUMORCE** [comment, marked as answer]
`scoop shim rm more`, you can also install `main/uutils-coreutils-lean` then adding shims you want: `scoop shim add foo "bar..."` or use them `uutils <tool> [args...]` directly.

the native more command has a higher priority, in general case:

```
❯ scoop shim list | Where-Object name -eq 'more'

Name Source           Alternatives IsGlobal IsHidden
---- ------           ------------ -------- --------
more uutils-coreutils              False    True

❯ where.exe more
C:\Windows\System32\more.com
C:\Users\humorce\scoop\shims\more.exe

❯ Get-Command more

CommandType     Name                                               Version    Source
-----------     ----                                               -------    ------
Application     more.com                                           10.0.2262… C:\Windows\system32\more.com

❯ Get-Command more.exe

CommandType     Name                                               Version    Source
-----------     ----                                               -------    ------
Application     more.exe                                           0.0.0.0    C:\Users\humorce\scoop\shims\more.exe
```

**Reply by eggbean** [reply]
Thanks
